src/components/KDateTimePicker/KDateTimePicker.vue

```diff
@@ -705,16 +703,23 @@ $margin: 6px;
       border: 0;
     }
 
+    // disabled day
+    .vc-day-content.is-disabled {
+      pointer-events: none;
+    }
```

Comment:
question: is this desired over a `not-allowed` cursor?

Reply:
If you think it's better to add `cursor: not-allowed` I'm fine with that, but then I suggest that I add it to both disabled days and disabled months